src/components/PopoverMenuMultiSelect/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { ListItemText } from '@mui/material';
+import { DropdownItem } from '../types';
+import Icon from '../Icon/Icon';
+import './styles.scss';
+import Popover from '../PopoverMenu/Popover';
+
+export type PopoverMenuMultiSelectProps = {
+  anchorElement: HTMLElement | null;
+  initialSelection: any[];
+  menuAlign?: 'left' | 'center' | 'right';
+  onChange: (selection: any[]) => void;
+  options: any[];
+  optionRenderer: (option: any) => string;
+  setAnchorElement: React.Dispatch<React.SetStateAction<HTMLElement | null>>;
```

Comment:
ok to make this a `(anchorEl: HTMLElement) => void` .. not a huge fan of exposing underling mechanics of how the anchor element is set